### DISCUSSION TITLE
I cannot see options for Autotypes in internal table creation

### DISCUSSION THREAD

**Discussion by LenWagner** [Budibase/budibase#12067] [Help] [answered]
**Describe the bug**
1) I cannot see options to chose from when creating a Budibase Internal Table like "CreatedBy"...
2) I cannot see any options for AutoTypes when creating a Column with AutoColumn checked

**Expected behavior**
Based on your Youtube Tutorials, I expected to create a table with options like "CreatedBy, UploadedBy, etc."
Also, when I created a table, I cannot see these options either when I want to add AutoID columns... 

<img width="457" alt="image" src="https://github.com/Budibase/budibase/assets/47855082/f77cee6e-07f1-4b92-a82d-31362d1d9612">
<img width="390" alt="image" src="https://github.com/Budibase/budibase/assets/47855082/28bcbb17-2ef2-44bf-b50f-55f8ff64b3c2">

**App Export**
If possible - please attach an export of your budibase application for debugging/reproduction purposes.

**Desktop (please complete the following information):**
 - OS: MAC OS
 - Browser SAFARI and FIREFOX

**Comment by ConorWebb96** [comment]
Hey @LenWagner,

Converting this to a discussion as its not a bug.

After selecting the auto column you should see a new field after selecting the auto column option. From here you should be able to select what you want e.g. `created by`, `updated at` etc. (I've attached a screenshot below to aid with this explanation)

I've also tested this on Safari and it works on that browser too.
<img width="386" alt="Screenshot 2023-10-16 at 07 31 35" src="https://github.com/Budibase/budibase/assets/126772285/67088c52-f1b8-47e5-bcef-51bf61542504">

I hope this helps.

**Reply by shurlakov** [reply]
Hi @ConorWebb96,

I'm experiencing the same issue. It worked as expected before updating to version 2.11.33. The issue is not reproducible on version 2.11.8, for example.

**Reply by ConorWebb96** [reply, marked as answer]
I've just upgraded to the latest you are right. I've written up a bug report [here](https://github.com/Budibase/budibase/issues/12068) with replication steps. It seems to have happened within the last minor 17 versions as I was on .16 before. This seems to be a regression.

Thanks for pointing this out!
